### PR TITLE
[T4A4][W11-B2]Tan Wang Leng

### PR DESCRIPTION
**Pull request by yamgent**

### CODE REVIEW

**Comment by yamgent** [comment]
Ready for review

**Comment by yamgent** [comment]
Further commits removed the magic "strings". Prefixes now resides in class as constants.

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -47,6 +48,20 @@ public TextUi(InputStream in, PrintStream out) {
...
+     */
+    public static String getPrintableString(Printable... printables) {
+        List<String> printableStrings = new ArrayList<String>();
+        
+        for (Printable p : printables) {
+            printableStrings.add(p.getPrintableString());
+        }
+                
+        return String.join(", ", printableStrings);
+    }
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java:65]: Good!

**Comment by cmkumar87** [comment]
@yamgent  good job! psl close PR

**Pull request closed without merging**